aws_tools/s3_handler.py

```diff
@@ -109,8 +131,8 @@ def get_file_contents(self, key, catch_exception=True):
     def get_json(self, key, catch_exception = True):
         if catch_exception:
             try:
-               return json.loads(self.get_file_contents(key))
-            except Exception:
+                return json.loads(self.get_file_contents(key))
+            except Exception as e:
```

Comment:
If you are just ignoring all exceptions, you just need `except:`, the rest does nothing.